dev/FHIR3-0-2-Geboortezorg/Zwangerschapskaart/PHR-Client/gbz-fhir3-0-2-izk-phr-1-2.xml

```diff
@@ -748,9 +748,9 @@
             <extension url="http://touchstone.aegis.net/touchstone/fhir/testing/StructureDefinition/testscript-assert-stopTestOnFail">
                <valueBoolean value="false"/>
             </extension>
-            <description value="Confirm that the response Bundle contains 4 Procedure resource(s). "/>
+            <description value="Confirm that the response Bundle contains 3 Procedure resource(s). "/>
```

Comment:
Moet het niet eigenlijk **Confirm that the response Bundle contains 3 or more Procedure resource(s).**
Mischien een beetje nitpicky